src/state/navigation/extensionsAtom.ts

```diff
@@ -324,6 +331,12 @@ const getExtensions = async (
           ) as ExtResource,
         };
 
+        if (extCustomComponentsEnabled) {
+          extResourceWithMetadata.data.customHtml =
+            currentConfigMap.data.customHtml;
```

Comment:
Maybe just add here this, and remove the previous copy-paste?
```suggestion
        if (extCustomComponentsEnabled) {
          extResourceWithMetadata.data.customHtml =
            currentConfigMap.data.customHtml || '';
```